inc/functions.php

```diff
@@ -69,14 +69,14 @@ function core_sitemaps_get_server() {
  *
  * @return array $sitemaps A list of registered sitemap providers.
  */
-function core_sitemaps_get_sitemaps() {
-	$core_sitemaps = core_sitemaps_get_server();
+function sitemaps_get_sitemaps() {
```

Comment:
That's an odd name if we'd want to have this as-is in core. What about `wp_get_sitemaps()`?

Reply:
good suggestion, I'll scan for any other "double" use of sitemaps in names.